Arm/Insts/DPR/Data_processing_one_source.lean

```diff
@@ -0,0 +1,143 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+-- REV, REV16, REV32
+
+import Arm.Decode
+import Arm.Insts.Common
+
+----------------------------------------------------------------------
+
+namespace DPR
+
+open Std.BitVec
+
+theorem shiftLeft_ge (x : Nat) (y : Nat) : x ≤ x <<< y := by
```

Comment:
I'd recommend putting `private` for lemmas you'd like to be local to this file. This is probably something that we ought to do for other files too at some point.